daemon/main.c

```diff
@@ -516,6 +516,58 @@ static void backwards_compatible_config() {
 
     config_move(CONFIG_SECTION_STATSD,  "enabled",
                 CONFIG_SECTION_PLUGINS, "statsd");
+
+    config_move(CONFIG_SECTION_GLOBAL,  "memory mode",
+                CONFIG_SECTION_DB,      "mode");
+
+    config_move(CONFIG_SECTION_GLOBAL,  "history",
+                CONFIG_SECTION_DB,      "retention");
+
+    config_move(CONFIG_SECTION_GLOBAL,  "update every",
+                CONFIG_SECTION_DB,      "granularity secs");
```

Comment:
If we do this change we also need to update:
 - the rest of the code in netdata/netdata (1845 matches)

```c
# this branch
deb-work nd_ktsaou ((cd11309b7...)) $ git grep "update.every" -- '*.c' '*.h' |  wc -l
1845
```

- documentation (114 matches in netdata/learn)

```cmd
learn (master) $ git grep 'update.every' | wc -l
114
```

And I'd suggest using "collect every".

Reply:
I reverted it back to `update every`